python/docs/schema/ValueType.adoc

```diff
@@ -25,7 +25,7 @@ a| `STRING` a| `_ValueType(is_writable=True, is_keyable=True, 4)`
 
 [source,python]
 ----
-static of(value_type: Object | Boolean | Long | Double | String | DateTime) -> ValueType
+static of(value_type: 0 | 1 | 2 | 3 | 4 | 5) -> ValueType
```

Comment:
Sphinx with Py3.11 for whatever reason uses the integer representation for the constants.

Reply:
That's not great - can we look into it to try to get the names back?

Reply:
These really are just integers. I have not found any issues about that change, and anything more in depth is out of scope of this PR.

Reply:
Since this method is meant for internal use only, we've decided to remove it from the doc generation altogether.